apps/librelingo_types/librelingo_types/data_types.py

```diff
@@ -78,6 +79,7 @@ class HunspellSettings(
     pass
 
 
+# pylint: disable=line-too-long
```

Comment:
i disable them here, because i didn't found an "elegant" solution for the docstring tests, if you have any i am eager to discover it!